.github/workflows/zosEndToEndTest.yml

```diff
@@ -0,0 +1,63 @@
+name: z/OS end-to-end Tests
+
+on:
+  pull_request:
```

Comment:
I guess this is for testing, but ultimately I don't think we should run that on each PR

Reply:
Yes, it's for testing